### DISCUSSION TITLE
not working on Venus.OS

### DISCUSSION THREAD

**Discussion by ronnies007** [node-red-contrib-themes/theme-collection#495] [General]
### Describe the problem

When u run Node-Red from within Venus.OS on a Raspberry Pi, the way described here wont work.
the installing part worked but I couldnt activate the dark-theme.
this took me days to find out, here is howto.

open this file with:
nano /usr/lib/node_modules/@victronenergy/node-red-contrib-victron/settings-default.js

change it so that the bottom lines look like this:
    // Customising the editor
    editorTheme: {
        page: {
            css: "/data/home/root/node_modules/@node-red-contrib-themes/theme-collection/themes/dark/dark.min.css"
        },
        //theme: "dark"
        theme: {
            dark: true
        }
        projects: {
            // To enable the Projects feature, set this value to true
            enabled: true
        },
        // disable the palettemanager feature
        palette: {
            editable: false
        }
    },
}

I tried some other themes but dark-theme is the only one working so far. 
Node-Red just wouldnt start with any other themes I tried.

maybe this is of some help to other raspberry pi users, who get ache from the bright white default-background, 
and ran into the same trouble.

### Theme Collection version

@node-red-contrib-themes/theme-collection@4.0.8

### Node-RED version

3.1.10

### Platform/O.S.

venus.os

**Comment by bonanitech** [comment]
I don't know Venus OS and can't help you with it, but your Node-RED version seems outdated. The latest Node-RED version is 4.0.2. You should upgrade.

If you really want to use Node-RED version 3.1.10, you must install the latest version of the theme collection that is compatible with that Node-RED version.

```
npm install @node-red-contrib-themes/theme-collection@v3.1
```

And, if you carefully read the [documentation](https://github.com/node-red-contrib-themes/theme-collection/blob/dev/README.md#usage), you will notice that your settings are incorrect.
